src/app/Http/Controllers/Auth/ResetPasswordController.php

```diff
@@ -32,6 +32,10 @@ public function __construct()
     {
         $this->middleware('guest');
 
+        if (!backpack_users_have_email()) {
+            abort(403, trans('backpack::base.no_email_column'));
```

Comment:
403 feels wrong to me.

Personally I think it's one of the 500s or a view that states "Your super intelligent administrator must be contacted to do this."

Reply:
True. Changing it to 501 (not implemented).